iphone/Classes/KrollBridge.m

```diff
@@ -1009,37 +988,63 @@ - (TiModule *)loadJavascriptText:(NSString *)data fromFile:(NSString *)filename
 	return module;
 }
 
-- (TiModule *)loadAsFile:(NSString *)path withContext:(KrollContext *)kroll
+- (KrollWrapper *)cachedLoadAsFile:(NSString *)path asJSON:(bool)json withContext:(KrollContext *)kroll
 {
-	// 1. If X is a file, load X as JavaScript text.  STOP
-	NSString *filename = path;
-	NSString *data = [self loadFile:filename];
+	// check cache first
+	if (modules != nil) {
+		KrollWrapper *module = [modules objectForKey:path];
+		if (module != nil) {
+			NSLog(@"[WARN][MODULE CACHE HIT] : %@", path);
+			return module;
+		}
+		NSLog(@"[WARN][MODULE CACHE MISS] : %@", path);
```

Comment:
Remove the `[WARN]` logs or make them `[TRACE]` / `[DEBUG]`. Should be done for all occurrences of this PR. Otherwise, der developers will get complain about internal / unrelated logs thrown to them.